client/i18n/locales/english/intro.json

```
@@ -3373,6 +3373,10 @@
       "exam-certified-full-stack-developer": {
         "title": "Certified Full Stack Developer Exam",
         "intro": ["Pass this exam to become a Certified Full Stack Developer."]
+      },
+      "workshop-recipe-ingredient-converter": {
+        "title": "Build a Recipe Ingredient Converter",
+        "intro": ["", ""]
       }
```

We should move this up with the other functional programming content. It looks like it would go somewhere around line 3050.

```suggestion
      },
      "workshop-recipe-ingredient-converter": {
        "title": "Build a Recipe Ingredient Converter",
        "intro": ["In the previous lecture videos, you learned the core concepts behind functional programming and currying.", "Now you will be able to apply what you have learned about currying and functional programming by building a recipe ingredient converter application."]
      }
```